test/e2e/route_util.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"net"
+	"regexp"
+	"strings"
+)
+
+type Route struct {
+	peerPodCIDR *net.IPNet
+	peerPodGW   net.IP
+}
+
+func getGatewayRoutes(data *TestData, isIPv6 bool) (routes []Route, err error) {
+	antreaGWName, err := data.GetGatewayInterfaceName(antreaNamespace)
+	if err != nil {
+		return nil, err
+	}
+	var cmd []string
+	nodeName := nodeName(0)
```

Comment:
It is not generic to assume the function always executed on `nodeName(0)`. But since it doesn't add new e2e test to leverage these utils now, extracting them is no longer necessary.